plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -320,6 +320,78 @@ fu_thunderbolt_device_get_attr_uint16(FuThunderboltDevice *self, const gchar *na
 	return (guint16)val;
 }
 
+static gboolean
+fu_thunderbolt_device_set_port_offline(FuUdevDevice *device)
+{
+	GError **error_local_offline = NULL;
+	GError **error_local_rescan = NULL;
+
+	if (!fu_udev_device_write_sysfs(device, "usb4_port1/offline", "1", error_local_offline)) {
```

Comment:
Are we hard coding usb4_port1?

Reply:
@djcampello  Yes currently there is just one port to update nvm and the other port shares the same so no need to update it , in later platforms EC needs to populate only the retimers which don't share the same NVM in that way we would grep the available usb4_portx and update nvm for all of them

Reply:
Closing this , Thank you!